template/src/main/twirl/ch.epfl.scala.index.views/project/install.scala.html

```diff
@@ -26,27 +26,46 @@
   </ul>
   <div class="tab-content">
     <div role="tabpanel" class="tab-pane active" id="sbt">
-      <pre>@release.sbtInstall</pre>
+      <pre id="copy-sbt" value="$@release.sbtInstall">@release.sbtInstall</pre>
```

Comment:
We could factorize it `@installtab(id: String, releaseInstall: String, active: Boolean, linkRef: String, linkText: String)`. Dunno if this any better though